io.elementary.Sdk.json

```diff
@@ -41,6 +41,32 @@
         }
     },
     "modules": [
+         {
+            "name": "gtk",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbroadway_backend=true",
+                "-Dinstalled_tests=true",
```

Comment:
we don't need the tests in the runtime

Reply:
I took the options from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/master/elements/sdk/gtk+-3.bst

Reply:
Yes, gnome-build-meta acts as project wide CI too so we want to make sure they are built there. We filter the produced binaries out from the runtime afterwards.